src/axolotl/utils/models.py

```diff
@@ -218,7 +218,9 @@ def load_tokenizer(cfg):
     LOG.debug(f"PAD: {tokenizer.pad_token_id} / {tokenizer.pad_token}")
     LOG.debug(f"UNK: {tokenizer.unk_token_id} / {tokenizer.unk_token}")
 
-    if cfg.chat_template:
+    if cfg.custom_chat_template:
+        tokenizer.chat_template = cfg.custom_chat_template
```

Comment:
similar to the chatml replacement, we could consider some sort of standardized string in the chat template that could be replaced based on the configuration.